types/style-inject/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for style-inject 0.3
+// Project: https://github.com/egoist/style-inject
+// Definitions by: Leon Si <https://github.com/leonzalion>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface StyleInjectOptions {
+	insertAt?: 'top';
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/style-inject/index.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.